src/components/calendar/event-utils.js

```diff
@@ -9,6 +9,11 @@ export function formatEvents(eventsArray) {
       title: event.name,
       start: event.startTime,
       end: event.endTime,
+      extendedProps: {
```

Comment:
This is definitely the cleaner way to do this in terms of passing the information into the event details